docs/src/start/connect-azure.md

```diff
@@ -29,7 +29,7 @@ See this [tutorial](connect-c8y.md), if you want to connect Cumulocity instead.
 See this [tutorial](connect-aws.md), if you want to connect AWS IoT instead.
 
 Before you try to connect your device to Azure IoT, you need:
-* Create a Azure **IoT Hub** in Azure portal as described [here](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-create-through-portal).
+* Create a Azure **IoT Hub** in Azure portal as described [here](https://learn.microsoft.com/en-us/azure/iot-hub/iot-hub-create-through-portal).
```

Comment:
Actually this one redirected me to https://learn.microsoft.com/en-us/azure/iot-hub/create-hub?tabs=portal

Reply:
So it seems that mlc only follows 1 redirect...

Reply:
I'll change the link to `https://learn.microsoft.com/en-us/azure/iot-hub/create-hub`...but I'll leave off the query parameter preferences (it is just controlling which code tab is opened on the page...but we don't really care about that).

Reply:
Resolved by 98f0b059b593f5ee8d827baf86890ee5a487bd89